framework/src/base/MooseApp.C

```diff
@@ -672,7 +672,16 @@ MooseApp::restore(std::shared_ptr<Backup> backup, bool for_restart)
 void
 MooseApp::setCheckUnusedFlag(bool warn_is_error)
 {
-  _enable_unused_check = warn_is_error ? ERROR_UNUSED : WARN_UNUSED;
+  /**
+   * _enable_unused_check is initialized to WARN_UNUSED. If an application chooses to promote
+   * this value to ERROR_UNUSED pragmatically prior to running the simulation, we certainly
```

Comment:
pragmatically -> programatically?

Reply:
Well, I'm always pragmatic 😉 .

I'll fix this really quickly. You can probably merge before testing is complete if it's just a typo fix.